site/downloads.html

```diff
@@ -182,9 +182,11 @@ <h3 id="link-with-spark">Link with Spark</h3>
 <h3 id="installing-with-pypi">Installing with PyPi</h3>
 <p><a href="https://pypi.org/project/pyspark/">PySpark</a> is now available in pypi. To install just run <code class="language-plaintext highlighter-rouge">pip install pyspark</code>.</p>
 
-<h3 id="convenience-docker-container-images">Convenience Docker Container Images</h3>
+<h3 id="installing-with-docker">Installing with Docker</h3>
 
-<p><a href="https://hub.docker.com/r/apache/spark-py/tags">Spark Docker Container images are available from DockerHub</a>, these images contain non-ASF software and may be subject to different license terms.</p>
+<p>Spark docker images are available from Dockerhub under the accounts of both <a href="https://hub.docker.com/r/apache/spark/">The Apache Software Foundation</a> and <a href="https://hub.docker.com/_/spark">Official Images</a>.</p>
+
+<p>Note that, these images contain non-ASF software and may be subject to different license terms. Please check their <a href="https://github.com/apache/spark-docker">Dockerfiles</a> to verify whether to verify whether they are compatible with your deployment.</p>
```

Comment:
It seems that we disobey the ASF release policies. https://incubator.apache.org/guides/distribution.html#docker

> docker pull apache/<project> should not install an artifact containing unapproved code.